src/libs/ascent/runtimes/flow_filters/ascent_runtime_utils.cpp

```diff
@@ -56,6 +62,39 @@ std::string output_dir(const std::string file_name)
   return output_path;
 }
 
+//-----------------------------------------------------------------------------
+
+bool check_dir_path_exists(const std::string file_path, int mpi_comms_id, conduit::Node &err_msg)
+{
+  int rank = 0;
+#ifdef ASCENT_MPI_ENABLED
+  MPI_Comm mpi_comm = MPI_Comm_f2c(mpi_comms_id);
+  MPI_Comm_rank(mpi_comm, &rank);
+#endif
+
+  int res = true;
```

Comment:
minor suggestion:
Use 1/0 b/c the type is an int. 

Using true and false is more readable, but it makes me think we are using bools, which we don't want to do for MPI calls.